xarray/core/dataarray.py

```diff
@@ -3430,6 +3432,301 @@ def pad(
         )
         return self._from_temp_dataset(ds)
 
+    def _calc_idxminmax(
```

Comment:
We've generally put these sorts of functions in `core/computation.py`, but I actually think it's fine here assuming others do

Reply:
Yeah let's move it to `computation.py`. `dataarray.py` is quite long

Reply:
Okay